curations/maven/mavencentral/org.bouncycastle/bcprov-jdk15on.yaml

```diff
@@ -24,6 +24,22 @@ revisions:
   '1.64':
     licensed:
       declared: MIT
+  '1.65.01':
+    licensed:
+      declared: MIT
+    sourceLocation:
+      name: bcprov-jdk15on
+      namespace: org.bouncycastle
+      provider: mavencentral
+      revision: '1.65.01'
+      type: sourcearchive
+      url: >-
+        https://search.maven.org/remotecontent?filepath=org/bouncycastle/bcprov-jdk15on/1.65.01/bcprov-jdk15on-1.65.01-sources.jar
+    urls:
```

Comment:
It looks like this part is not curate-able. Can I delete this so I can merge the rest?

Reply:
sure, the documentation what exactly can be curated and which fields are added by some bot isn't very clear and the UI doesn't allow to specify a source artefact from maven central but only a git repository